### PR TITLE
Complete Iteration 1: Austin Andrade

### PR DESCRIPTION
**Pull request by austinandrade**
## All Submissions:

* [x] Merge the target branch into the PR branch. Fix any conflicts that might appear.

### New Feature Submissions:

* [x] Have you added an explanation of what your changes do?
----------------------------------------------------------

Completed up Through Iteration 1. Let me know if I made any mistakes and if the code looks good!

### CODE REVIEW

File: app/models/game_company.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class GameCompany < ApplicationRecord
+  has_many :video_games, -> { order(created_at: :desc) }
```
Comment (georgehwho) [inline app/models/game_company.rb:2]: i think if it's already ordered here you don't have to order it inside of your app/view/game_companies/index

File: app/views/game_companies/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<p><a href="/game_companies/">List of Game Companies</a></p>
+<p><a href="/video_games/">List of Video Games</a></p>
+<br>
+
+<h1>All Game Companies</h1>
+
+<% @game_companies.sort_by(&:created_at).reverse!.each do |company| %>
```
Comment (georgehwho) [inline app/views/game_companies/index.html.erb:7]: see note in game company model

**Review by georgehwho — APPROVED** [review]
All in all, very nice!

File: app/models/game_company.rb (hunk 1 above)
Comment (austinandrade) [reply app/models/game_company.rb:2]: @georgehwho I think I got ahead of the game or even added an unnecessary feature, but that is actually supposed to be ordering the video games that are associated with each game company by their own creation times. I thought maybe it'd come in handy later, who knows though 🤷🏽‍♂️

**Pull request merged**